ci-best-practices.md

```diff
@@ -0,0 +1,29 @@
+#### CI should...
+
+- be "as a service".
+- only push 1 check to PR's in Github.

```

Comment:
This (and the `make ci` target) seems to be in opposition to projects "building" in multiple independently-executable steps